src/actions/contactsActions.js

```diff
@@ -77,16 +77,25 @@ export const syncContactAction = (userId: string) => {
       user: { data: { walletId } },
       contacts: { data: contacts },
       accessTokens: { data: accessTokens },
+      connectionIdentityKeys: { data: connections },
     } = getState();
 
-    const accessToken = accessTokens.find(token => token.userId === userId);
-    if (!accessToken) return;
-
-    const userInfo = await api.userInfoById(userId, {
-      walletId,
-      userAccessKey: accessToken.myAccessToken,
-      targetUserAccessKey: accessToken.userAccessToken,
-    });
+    let userInfo;
+    const connection = connections.find((conn: ConnectionIdentityKey) => conn.targetUserId === userId);
```

Comment:
This action suppose to call the api in order to sync the userinfo, we shouldn't just grab the cached data from the connections

Reply:
the data of connectionIdentityKeys is updated on every login or people screen refresh, the backend sends the map with the latest info of the contacts.

Reply:
👍